tests/reftests/conflict-4373.test

```diff
@@ -1,10 +0,0 @@
-c1d23f0e
-### OPAMVAR_enable_ocaml_beta_repository=1 opam switch create --fake 4.12.0
-[ERROR] No compiler matching `4.12.0' found, use `opam switch list-available' to see what is available, or use `--packages' to select packages explicitly.
-# Return code 5 #
-### opam install --show expect.0.0.6
-[ERROR] No switch is currently set. Please use 'opam switch' to set or install a switch
-# Return code 50 #
-### opam install --show ppx_expect.v0.14.0
-[ERROR] No switch is currently set. Please use 'opam switch' to set or install a switch
-# Return code 50 #
```

Comment:
Note for developers: this test was included in https://github.com/ocaml/opam/pull/4504 for the same purpose but has never tested anything